qemu/tests/qemu_guest_agent.py

```diff
@@ -3423,7 +3425,14 @@ def _result_check(rsult_qga, rsult_guest):
             date_group = re.search(r"driverdate.*\:\s(\d{4})(\d{2})(\d{2})",
                                    driver_info_guest, re.I).groups()
             driver_date_guest = "-".join(date_group)
-            _result_check(driver_date, driver_date_guest)
+            if main_qga_ver == "5":
+                driver_date_guest_timearray = time.strptime(driver_date_guest,
+                                                            "%Y-%m-%d")
+                driver_date_guest = time.mktime(driver_date_guest_timearray)
+                if abs(driver_date_guest - int(driver_date/1000000000)) > 86400:
+                    _result_check(driver_date, driver_date_guest)
```

Comment:
I don't really understand here, `_result_check` want to make sure `driver_date` = `driver_date_guest`, right? Then what is the meaning of `abs(driver_date_guest - int(driver_date/1000000000)) > 86400`? If it is not greater than 86400, then you check nothing? So it only fail when `abs(driver_date_guest - int(driver_date/1000000000)) > 86400` but `driver_date_guest` != `driver_date`?

Reply:
The new change of expression of driver_time is accurate to the minute and second, but you know there are some time difference between guest&qga, sort of following result:
04:49:11 DEBUG| [stdout] 1613692800000000000 ----> qga
04:49:11 DEBUG| [stdout] 2021-02-19 ----> guest
so I should convert the guest results. but there are still some difference with qga, but I found that they can't be 24 hours apart, so I set 86400 as the safe gap conditions.

Reply:
@6-dehan Thanks for the clear demonstration, while if so, I suppose _result_check(driver_date, driver_date_guest) is meaningless for new versions, `> 86400` or `else` should be the check point to determine pass or fail.

Reply:
@kimi1978wy Call for you review on this, thanks.

Reply:
okay, actually the reason I also keep _result_check() function there because if the difference is greater than 86400, it is equivalent to the situation where qga and guest are not equal. anyway, I'd like to to absorb your suggestion and directly judge whether it passes or fails.

Reply:
> @6-dehan Thanks for the clear demonstration, while if so, I suppose _result_check(driver_date, driver_date_guest) is meaningless for new versions, `> 86400` or `else` should be the check point to determine pass or fail.

Yes, you're right. demeng modified this part, you can review it again now